frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -956,7 +956,9 @@ class Worksheet extends React.Component {
             editor.$blockScrolling = Infinity;
             editor.session.setUseWrapMode(false);
             editor.setShowPrintMargin(false);
-            editor.session.setMode('ace/mode/markdown');
+            editor.session.setMode('ace/mode/markdown', function() {
+                editor.session.$mode.blockComment = { start: '//', end: '' };
```

Comment:
Adding a space makes it feel more natural and is more in line with what other IDEs do.

```suggestion
                editor.session.$mode.blockComment = { start: '// ', end: '' };
```

Reply:
While this is true, if someone manually adds "//" at the beginning, then using '// ' will make it "// //" instead of uncommenting it. If that makes sense

Reply:
I see. I think we shouldn't use `blockComment` here. Can we use `lineCommentStart` instead?

See https://github.com/ajaxorg/ace/blob/9b5b63d1dc7c1b81b58d30c87d14b5905d030ca5/tool/tmlanguage.js#L655-L661

Reply:
Hmmm, but if you select multiple line, it triggers blockComment I think. Moreover, the current behavior is to add \<!-- -->, not any of those that you highlighted

Reply:
Ok, seems more complicated than I initially thought. The current "//" behavior is fine then.